js/client-mainmenu.js

```diff
@@ -1487,5 +1500,151 @@
 			return buf;
 		}
 	});
+	//team select event that triger when user pick a team
+	var on_team_select=new Event("onselectteam_event");
+
+	function attatchEventToElement(element_name,event_name,even_lisener){
+		let element=document.querySelector(element_name)
+		element.addEventListener(event_name,function(e){even_lisener()});
+	}
+	function trigerEvent(element_name,event_obj){
+		let element=document.querySelector(element_name)
+		element.dispatchEvent(event_obj)
+	}
+	//creat observer for log in fourm to check when user log in 
+	function createloginFourmObserver(){
```

Comment:
```suggestion
	function createLoginFormObserver(){
```
Example spelling fix.